lib/modules/manager/poetry/__snapshots__/extract.spec.ts.snap

```diff
@@ -377,31 +377,14 @@ exports[`modules/manager/poetry/extract extractPackageFile() extracts multiple d
       "datasource": "pypi",
       "depName": "dep3",
       "depType": "dependencies",
-      "managerData": {
-        "nestedVersion": true,
-      },
       "skipReason": "path-dependency",
     },
     {
-      "currentValue": "",
       "datasource": "pypi",
       "depName": "dep4",
       "depType": "dependencies",
-      "managerData": {
-        "nestedVersion": false,
-      },
       "skipReason": "path-dependency",
     },
-    {
-      "currentValue": "",
-      "datasource": "pypi",
-      "depName": "dep5",
-      "depType": "dependencies",
-      "managerData": {
-        "nestedVersion": false,
-      },
-      "versioning": "pep440",
-    },
```

Comment:
This is result from `dep5 = {}` string parsing. I think it's an error to produce value without `skip-reason` for it.

Reply:
If we detect the dependency then can we extract it, even if we skip it? This is generally intuitive for users

Reply:
Okay